case/tube.F90

```diff
@@ -75,22 +75,26 @@ subroutine InitSystem
     call ReadRestart(restart_file)
 
     ! Reference cells
-    allocate(rbcRefs(2))
+    allocate(rbcRefs(3))
 
     if (nrbc > 0) then
       radEqv = 1.
       nlat0 = rbcs(1)%nlat0
       
       rbcRef => rbcRefs(1)
       call RBC_Create(rbcRef, nlat0)
-      call RBC_MakeBiconcave(rbcRef, radEqv)
+      call RBC_MakeBiconcave(rbcRef, radEqv) 
       call RBC_ComputeGeometry(rbcRef)
 
       rbcRef => rbcRefs(2)
       call RBC_Create(rbcRef, nlat0)
       call RBC_MakeSphere(rbcRef, radEqv)
       call RBC_ComputeGeometry(rbcRef)
 
+      rbcRef => rbcRefs(3)
+      call ReadRBCPlain('Input/SickleCell.dat', rbcRef)
```

Comment:
so plain is sickle cell? what happens if the number of points on the sickle cell `nlat` and `nlon` doesn't match the values the user inputs?

Reply:
the sickle cell's nlat/nlon in this case will just be the same as the nlat/nlon of the exported cell-shape. So, even if the user changes the input nlat0, the sickle cell disregards that and uses the model's data from the `Input/SickleCell.dat` file.